Sample_Convo_html/email1.html

```
@@ -0,0 +1,806 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="https://static.zdassets.com/classic/assets/email-f04bd8b49240e055ba602c6b7d08da042d046e308b92a11fdb390398af4e7379.css" />
+    <script src="https://static.zdassets.com/classic/assets/email-f5e4e90b0480c4bef39a097a3d56f30c5dc8c885eb60a5437bce6838334f5e1d.js"></script>
+  </head>
```

HTMLHint (1.5.0)

[error] <title> must be present in <head> tag.

(title-require)